server/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -51,13 +52,11 @@
 
     public static final boolean ADJUST_PURE_NEGATIVE_DEFAULT = true;
 
-    private static final String MUSTNOT = "mustNot";
-    private static final String MUST_NOT = "must_not";
-    private static final String FILTER = "filter";
-    private static final String SHOULD = "should";
-    private static final String MUST = "must";
-    private static final ParseField DISABLE_COORD_FIELD = new ParseField("disable_coord")
-            .withAllDeprecated("disable_coord has been removed");
+    private static final ParseField MUSTNOT = new ParseField("mustNot");    // TODO deprecate?
```

Comment:
I think so!

Reply:
I think I'll do this in a followup to get more eyes on it.